modules/exploits/multi/http/churchinfo_upload_exec.rb

```diff
@@ -0,0 +1,183 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'ChurchInfo v.1.3.0 Authenticated RCE',
+        'Description' => %q{
+          This module exploits the logic in the CartView.php page when crafting a draft email with an attachment.
+          By uploading an attachment for a draft email, the attachment will be placed in the /tmp_attach/ folder of the
+          ChurchInfo web server, which is accessible over the web by any user. By uploading a PHP attachment and
+          then browsing to the location of the uploaded PHP file on the web server, arbitrary code
+          execution as the XXX user can be achieved.
```

Comment:
Is `XXX` a defined user or was this expected to be updated?  Is this run in the context of the `web server` account maybe?  Please also update the `md` file if this does need to change.

Reply:
This was from the changes @gwillcox-r7 added in his comments, I've updated this in the latest version of code